pydicom/filewriter.py

```diff
@@ -938,47 +976,39 @@ def dcmwrite(filename, dataset, write_like_original=True):
     else:
         fp = DicomFileLike(filename)
 
-    # if we want to write with the same endianess and VR handling as
-    # the read dataset we want to preserve raw data elements for
-    # performance reasons (which is done by get_item);
-    # otherwise we use the default converting item getter
-    if dataset.is_original_encoding:
-        get_item = Dataset.get_item
-    else:
-        get_item = Dataset.__getitem__
-
     try:
         # WRITE FILE META INFORMATION
         if preamble:
             # Write the 'DICM' prefix if and only if we write the preamble
             fp.write(preamble)
             fp.write(b'DICM')
 
+        tsyntax = None
         if dataset.file_meta:  # May be an empty Dataset
             # If we want to `write_like_original`, don't enforce_standard
             write_file_meta_info(fp, dataset.file_meta,
                                  enforce_standard=not write_like_original)
+            tsyntax = getattr(dataset.file_meta, "TransferSyntaxUID", None)
+
+        if (tsyntax == DeflatedExplicitVRLittleEndian):
+            # See PS3.6-2008 A.5 (p 71)
```

Comment:
should be PS3.5

Reply:
Indeed! I could even update to a modern edition.
Confesssion: I stole this comment from https://github.com/pydicom/pydicom/blob/5098c9147fadcb3e5918487036867931435adeb8/pydicom/filereader.py#L713
What's your policy on unrelated changes in PRs? If it's convenient for you, I can update that reference too. Or I could submit a separate PR.

Reply:
Ah, nice :) Yes, go ahead and correct it, it is only a typo. BTW, I don't mention year and page in these references at all. As they don't change the numbering in new editions, and the chapters are usually small, I think this is sufficient.

Reply:
Page reference is outdated though

Reply:
Both sites updated in their own commits (one compensating, one just new) . Years and page numbers dropped. Thanks.